### ISSUE TITLE
Accuracy editor needs "quit without saving:

### ISSUE THREAD

**Issue by JohnDenker** [barbeau/gpstest#698]
This is either a major suggestion or a minor bug report.

For several reasons, the editor for entering lat/lon/elevation on the Accuracy page needs a button for "quit without saving".

Use case 1: Sometimes I key in a wrong number, and want to start over.

Use case 2: Clicking on the map relocates the reference point. Sometimes this is a useful feature, and sometimes it's a disaster, since it erases desirable numbers.

The workaround is to exit the app completely and get back in, but it took me several disasters to figure that out.

Bottom line: The escape route needs to be clearly indicated as part of the UI.

See also next issue.
